tests/setup.js

```
@@ -0,0 +1,29 @@
+  debug: vi.fn(() => Promise.resolve()),
+  info: vi.fn(() => Promise.resolve()),
+  warn: vi.fn(() => Promise.resolve()),
+  error: vi.fn(() => Promise.resolve()),
+}));
+
+// Stub window.__TAURI__ at module level so that pomodoro-timer.js can destructure
+// `window.__TAURI__.core` at import time (before any beforeAll hook runs).
+globalThis.__TAURI__ = {
+  core: {
+    invoke: vi.fn((cmd) => {
+      if (cmd === "load_tasks") return Promise.resolve([]);
```

ESLint

[error] Expected { after 'if' condition.

(curly)